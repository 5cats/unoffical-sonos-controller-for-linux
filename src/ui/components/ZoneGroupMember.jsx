import React from 'react';

export function ZoneGroupMember(props) {
    return (
        <li>
            <span>{props.member.ZoneName}</span>
        </li>
    );
}

export default ZoneGroupMember;
