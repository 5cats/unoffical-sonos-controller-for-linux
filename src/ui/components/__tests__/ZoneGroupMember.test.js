import React from 'react';
import { ZoneGroupMember } from '../ZoneGroupMember';
import { render } from 'enzyme';

describe('ZoneGroupMember', () => {
    it('matches snapshot', () => {
        const props = {
            member: {
                name: 'My Name'
            }
        };
        const context = render(<ZoneGroupMember {...props} />);
        expect(context).toMatchSnapshot();
    });
});
