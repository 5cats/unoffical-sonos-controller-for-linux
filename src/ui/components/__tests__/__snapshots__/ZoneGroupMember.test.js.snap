// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ZoneGroupMember matches snapshot 1`] = `
<li>
  <span>
    My Name
  </span>
</li>
`;
